syntax="proto3";

// Build grpc for cpp and golang:
//   protoc --grpc_out=[CPP_GRPC_DIR] \
//          --plugin=protoc-gen-grpc=[PATH_TO_grpc_cpp_plugin] \
//          --cpp_out=[CPP_DIR] \
//          --go_out=plugins=grpc:[GO_DIR] price.proto

package price;

service Pricer {
    rpc GetPrice (ItemID) returns (ItemPrice) {}
}

message ItemID {
    int64 ID = 1;
}

message ItemPrice {
    float price = 1;
}
